test/prototype_transforms_kernel_infos.py

```diff
@@ -573,12 +588,16 @@ def transform(bbox, affine_matrix_, format_):
         out_bbox = F.convert_format_bounding_box(
             out_bbox, old_format=datapoints.BoundingBoxFormat.XYXY, new_format=format_, inplace=True
         )
+
+        # new_spatial_size = F._geometry._compute_affine_output_size(FIXME, w=spatial_size_[1], h=spatial_size_[0])
+        new_spatial_size = spatial_size_
+        out_bbox = F.clamp_bounding_box(out_bbox, format=format_, spatial_size=new_spatial_size)
```

Comment:
We need to compute the new spatial size here. Otherwise clamping will distort. As is this works ok-ish for all kernels that don't change the size like affine or rotate, but breaks down for crop, pad, resize, ...